### DISCUSSION TITLE
为啥不用 ccxt 集成gateway呢？

### DISCUSSION THREAD

**Discussion by cloudseasail** [veighna-global/vnpy_evo#7] [Ideas]
是否已经有类似的插件了？

**Comment by tingle2008** [comment]
我也发现了，ccxt pro 才支持wss 订阅,  下半年 ccxt pro免费了应该可以用ccxt 作为基础实现gateway

**Reply by cloudseasail** [reply]
请问在哪看到ccxt pro会免费的消息的？ 
至少RestAPI 可以用啊，不然很多交易所完全用不了

**Reply by tingle2008** [reply]
<img width="513" alt="image" src="https://user-images.githubusercontent.com/936995/174465634-df15e0aa-329e-42c9-9173-931ff2b3ba1a.png">
